### PR TITLE
Optimize auth flow refinements

### PR DESCRIPTION
**Pull request by mcclowes**
# Description

Optimize auth flow refinements

- Improving navigation
- Sort headers
- Read next
- Clarify monitoring flow logic if not using Link

### CODE REVIEW

File: docs/auth-flow/optimize/monitor-auth-flow.md [hunk 1]
```diff
@@ -9,15 +9,31 @@ We equip you with tools to monitor your auth flow. In the [Codat Portal](https:/
...
 If an integration category is marked as optional in **Settings > Auth flow > Link**, skipping that step increases the auth success measure of the funnel.
 :::
 
-Finally, visualize your stats of auth flow completion using the **graph**. It represents the ratio of the number of completed auth journeys to the number of started auth flows within a given week, expressed as a percentage. 
+:::info Using your own auth flow?
+
+If you've built your own solution, the **Began flow** step will be backfilled 
+:::
+
+### Completion % over time
```
Comment (pzaichkina) [inline docs/auth-flow/optimize/monitor-auth-flow.md:26]:
```suggestion
### Completion rate over time
```

File: docs/auth-flow/optimize/monitor-auth-flow.md [hunk 2]
```diff
@@ -9,15 +9,31 @@ We equip you with tools to monitor your auth flow. In the [Codat Portal](https:/
...
 
 :::info Optional steps
 
 If an integration category is marked as optional in **Settings > Auth flow > Link**, skipping that step increases the auth success measure of the funnel.
 :::
 
-Finally, visualize your stats of auth flow completion using the **graph**. It represents the ratio of the number of completed auth journeys to the number of started auth flows within a given week, expressed as a percentage. 
+:::info Using your own auth flow?
+
+If you've built your own solution, the **Began flow** step will be backfilled
```
Comment (pzaichkina) [inline docs/auth-flow/optimize/monitor-auth-flow.md:23]: is there anything we can add here? backfilled with what data? backfilled to how long backwards? etc

File: docs/auth-flow/optimize/optimize-the-connection-journey.md [hunk 3]
```diff
@@ -40,9 +44,15 @@ For further resources on good design practices and some inspiration, Codat recom
   </a>
   : the world’s largest mobile app design reference library
 
-:::info
+:::info Think about your usecase
```
Comment (pzaichkina) [inline docs/auth-flow/optimize/optimize-the-connection-journey.md:47]:
```suggestion
:::info Think about your use case
```

File: docs/auth-flow/optimize/optimize-the-connection-journey.md [hunk 4]
```diff
@@ -40,9 +44,15 @@ For further resources on good design practices and some inspiration, Codat recom
...
 
-Note whilst these recommendations apply universally, some examples are more relevant for specific use cases, such as lending and underwriting, insights, or reconciliation.
+Whilst these recommendations apply universally, some examples are more relevant for specific use cases, such as lending and underwriting, insights, or reconciliation.
```
Comment (pzaichkina) [inline docs/auth-flow/optimize/optimize-the-connection-journey.md:49]:
```suggestion
While these recommendations apply universally, some examples are more relevant for specific use cases, such as lending and underwriting, insights, or reconciliation.
```

File: docs/auth-flow/optimize/monitor-auth-flow.md (hunk 2 above)
Comment (mcclowes) [inline docs/auth-flow/optimize/monitor-auth-flow.md:23]:
```suggestion
If you've built your own solution, we aren't able to collect data for the **Began flow** step. Instead, we'll estimate that step in the funnel based on the number of companies that go to the next step. All the other data points should work as expected.
```

**Review by pzaichkina — APPROVED** [review]

**Pull request merged**